Fail to build cmrt if libva is installed at custom location

Try to build media stack as follows:

```
export CUSTOM_DIR=/path/to/custom/location  # not /usr or /usr/inlcude
cd libva
./autogen.sh --prefix=$CUSTOM_DIR
make && make install

export PKG_CONFIG_PATH=$CUSTOM_DIR/lib/pkgconfig
mkdir -p build && cd build
cmake ../media-driver \
  -DMEDIA_VERSION="2.0.0" \
  -DBUILD_ALONG_WITH_CMRTLIB=1 \
  -DBS_DIR_GMMLIB=`pwd`/../gmmlib/Source/GmmLib/ \
  -DBS_DIR_COMMON=`pwd`/../gmmlib/Source/Common/ \
  -DBS_DIR_INC=`pwd`/../gmmlib/Source/inc/ \
  -DBS_DIR_MEDIA=`pwd`/../media-driver \
  -DCMAKE_INSTALL_PREFIX=$CUSTOM_DIR \
  -DCMAKE_INSTALL_LIBDIR=$CUSTOM_DIR/lib \
  -DINSTALL_DRIVER_SYSCONF=OFF \
  -DLIBVA_DRIVERS_PATH=$CUSTOM_DIR/dri
make
```
You will see the following error:
Scanning dependencies of target igfxcmrt
```
make[2]: Leaving directory `/data/dvrogozh/git/github/build'
make[2]: Entering directory `/data/dvrogozh/git/github/build'
[  0%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/share/cm_printf_host.cpp.o
[  1%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/share/cm_rt_helpers.cpp.o
[  1%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/share/cm_avs_state_msg_ex.cpp.o
[  1%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/hardware/cm_device.cpp.o
[  1%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/hardware/cm_device_export.cpp.o
[  2%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/hardware/cm_surface_manager.cpp.o
[  2%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/hardware/cm_timer.cpp.o
[  2%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/hardware/cm_queue.cpp.o
[  2%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/__/agnostic/hardware/cm_perf_statistics.cpp.o
[  2%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/hardware/cm_device_os.cpp.o
[  3%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/hardware/cm_surface_manager_os.cpp.o
[  3%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/hardware/cm_device_export_os.cpp.o
[  3%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/hardware/cm_performance.cpp.o
[  3%] Building CXX object cmrtlib/linux/CMakeFiles/igfxcmrt.dir/hardware/cm_timer_os.cpp.o
[  4%] Linking CXX shared library igfxcmrt64.so
/usr/bin/ld: cannot find -lva
collect2: error: ld returned 1 exit status
make[2]: *** [cmrtlib/linux/igfxcmrt64.so] Error 1
make[2]: Leaving directory `/data/dvrogozh/git/github/build'
make[1]: *** [cmrtlib/linux/CMakeFiles/igfxcmrt.dir/all] Error 2
make[1]: Leaving directory `/data/dvrogozh/git/github/build'
make: *** [all] Error 2
make: Leaving directory `/data/dvrogozh/git/github/build'
```

--- a/cmrtlib/linux/CMakeLists.txt
+++ b/cmrtlib/linux/CMakeLists.txt
@@ -63,18 +63,11 @@ else()
     if(NOT LIBVA_FOUND)
         pkg_check_modules(LIBVA REQUIRED libva>=1.0.0)
     endif()
-    if(NOT LIBVA_X11_FOUND)
-        pkg_check_modules(LIBVA_X11 REQUIRED libva-x11>=1.0.0)
-    endif()
-    if(LIBVA_FOUND AND LIBVA_X11_FOUND)
+    if(LIBVA_FOUND)
         include_directories(BEFORE ${LIBVA_INCLUDE_DIRS})
-        include_directories(BEFORE ${LIBVA_X11_INCLUDE_DIRS})
+        link_directories(${LIBVA_LIBRARY_DIRS})
     endif()
 endif()
-
-
-include_directories(${MDF_OPTION__LIBVA_INCLUDE_DIR})
-link_directories(${MDF_OPTION__LIBVA_LIB_DIR})
 
 # Set up the source files
 set(CMRT_SOURCES "")

